dev-tools/deploy_railway.py

```
@@ -0,0 +1,202 @@
+import time
+from pathlib import Path
+
+# We're already in the backend directory
+backend_dir = Path(__file__).parent
+sys.path.insert(0, str(backend_dir))
+
+def run_command(cmd, cwd=None, check=True):
+    """Run a command and return the result"""
+    print(f"🔧 Running: {cmd}")
+    try:
+        result = subprocess.run(
```

Ruff (0.14.0)

`subprocess` call with `shell=True` identified, security issue

(S602)